test/functional/JIT_Test/src/jit/test/tr/BNDCHKImplicitNull.java/BNDCHKImplicitNullTest.java

```diff
@@ -0,0 +1,31 @@
+package jit.test.tr.BNDCHKImplicitNull;
+
+import org.testng.annotations.Test;
+import org.testng.AssertJUnit;
+
+@Test(groups = { "level.sanity","component.jit" })
+public class BNDCHKImplicitNullTest {
+
+  static class C {
+		private int idx = 1;
+		private double[] arr;
+
+		public void shouldThrowNullPointerException() {
+			for (int i = 0; i < 1; ++i) {
+				double x = arr[idx];
+			}
+		}
+	}
+
+	@Test
```

Comment:
Indentation?

Reply:
I'm not sure what happened here 🤔, it looks fine on my editor but when I push, this indentation is displayed instead.

Reply:
Fixed